### PR TITLE
KIN-396 incorrect error message on login

### PR DESCRIPTION
**Pull request by FernandoJoachin**
[KIN-396](https://loopcrack.atlassian.net/browse/KIN-396) Small [Powered by Pull Request Badge](https://pullrequestbadge.com/?utm_medium=github&utm_source=loopcrack-org&utm_campaign=badge_info)

### Problem

When a non-existent email is entered, the unconfirmed account alert is displayed

### Solution

Modify validations so they display properly.

### Requirements checklist

- [x] Do not show the unconfirmed account alert when a non-existent account is entered.
- [x] Show an appropriate alert for when a non-existent account is entered.

### Testing

- [x] Enter a non-existent email and verify that the correct alert is displayed
- [x] Enter an email that exists but is not confirmed and verify that the correct alert is displayed.
- [x] Enter an email that exists, that is confirmed but your password is incorrect and verify that the correct alert is displayed

[KIN-396]: https://loopcrack.atlassian.net/browse/KIN-396?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by CarlosJCD — CHANGES_REQUESTED** [review]

File: app/Validation/LoginValidation.php [hunk 1]
```diff
@@ -2,50 +2,63 @@
...
+        ],
+        'password' => [
+            'required' => 'La contraseña es obligatoria',
         ],
-        "password" => [
-            "required" => "La contraseña es obligatoria",
-        ]
     ];
+    protected $invalid_password_message    = 'Contraseña inválida. Por favor, intenta de nuevo.';
+    protected $invalid_account_message     = 'Correo electrónico no válido. Por favor, verifica el email proporcionado.';
```
[the remark this answers is not in the document]
Comment (CheloChidoone) [reply app/Validation/LoginValidation.php:23]: You are right @CarlosJCD it should be the same message just saying that either the password or email was invalid and not specifying, make this change please @FernandoJoachin

[the remark this answers is not in the document]
Comment (LUGAMAFE) [reply app/Validation/LoginValidation.php:23]: Yes, that's right, we shouldn't give the user hints about what is wrong.

**Review by LUGAMAFE — COMMENTED** [review]
Same comment that @CarlosJCD

**Review by CarlosJCD — APPROVED** [review]
Thanks for the changes!

**Review by Mauriciocr207 — APPROVED** [review]

**Pull request merged**